app/models/agents/imap_folder_agent.rb

```diff
@@ -54,6 +54,8 @@ class ImapFolderAgent < Agent
 
       Set `mark_as_read` to true to mark found mails as read.
 
+      Set `include_rfc822` to true to include a raw unencoded mail in each created event.
```

Comment:
Maybe swap `rfc822` and `raw` in the option name and description, `include_raw_data` and `to true to include the rfc822 formatted mail in each created event`? But I don't feel strongly as I am familiar how common the terms are.